src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -120,6 +123,39 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
             <span className="input-error">Password is required</span>
           )}
         </div>
+        <div className="input-group">
+          <input
+            name="date_of_birth"
+            className="input-block input-centered"
+            type="date"
+            ref={register}
```

Comment:
Here you only use `register`, but below you use `register({ require: true })`

Yet in the error message for `date_of_birth`, you're checking for a `required` error.

Is that correct? From reading the code, I would have expected this to also pass through the `required: true`, though i'm not familiar with the library semantics.

Reply:
Just added required true, it's there on the code field so should be on dob too.